components/pages/explorer/RepoTable.tsx

```diff
@@ -225,6 +230,24 @@ const getTableStyle = (theme: DMSThemeInterface) => css`
   }
 `;
 
+const VariantsFileIdCell = (props: any) => {
+  const { NEXT_PUBLIC_BASE_PATH } = getConfig();
+  const value = get(props, "original.occurrence.hits.edges[0].node.file.object_id");
```

Comment:
For some reason this field doesn't seem to have the "value" prop populated with the value of the cell, so I had to do some digging to get it. Not sure if this is related to the way it's indexed (since it's a nested field?), but this works anyway

Reply:
Kind of weird, I dont know the details on this offhand so I'm glad you found a working solution.